.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+inherit_gem:
+  salsify_rubocop: conf/rubocop.yml
+
+AllCops:
+  TargetRubyVersion: 2.4
```

Comment:
Should we go up to 2.5 to match rails 6?

Reply:
The gem currently requires Ruby >= 2.4 and Rails >= 4.2. I don't want to change that as part of this work but at some point we should drop Ruby 2.4/Rails 4.2.